index.html

```diff
@@ -9,6 +9,7 @@
     <script src="vendors/jquery/jquery.min.js"></script>
     <script src="vendors/materialize/js/materialize.min.js"></script>
     <script src="components/angular/angular.min.js"></script>
+    <script src="components/angular/angular-route.min.js"></script>
```

Comment:
So you can remove the angular-route.js file as you are including the minified version here 👍